src/ec/curve25519/ops.rs

```diff
@@ -63,4 +82,18 @@ impl Point {
             z: [0; ELEM_LIMBS],
         }
     }
+
+    pub fn to_bytes(&self) -> [u8; ELEM_LEN] {
+        let mut bytes = [0u8; ELEM_LEN];
+        unsafe { GFp_x25519_ge_tobytes(&mut bytes, self); }
+
+        bytes
+    }
+}
+
+extern {
+    fn GFp_ge_p3_tobytes(s: &mut [u8; ELEM_LEN], h: &ExtPoint);
```

Comment:
Note the change of signature, explained in the commit message.

These functions will be oxidized next (they are callers of `fe_invert()`, which is the motivation). Maybe we should add a separate type alias for encoded/compressed points, not just their byte length?